superduperdb/backends/base/query.py

```diff
@@ -145,6 +145,33 @@ def _set_db(r, db):
             parts.append((part, part_args, part_kwargs))
         self.parts = parts
 
+    def listeners(self, ):
+        listeners = self.db.show('listener')
+        vector_indices = self.db.show('vector_index')
+        dependencies = []
+        def _check_query_match(listener, query):
+            if (
+                listener.select.table_or_collection.identifier
```

Comment:
I think that `table_or_collection` isn't needed anymore. 
I think it's just `select.table`. (We can actually deprecate `table_or_collection`.)

Reply:
ok